packages/cinterop/build.gradle.kts

```diff
@@ -792,3 +749,12 @@ tasks.named("clean") {
         delete(project.file(".cxx"))
     }
 }
+
+// Generate an object holding the current release version, to be used by the JVM SoLoader path construction
+buildkonfig {
```

Comment:
This is already being generated by `cinterop` here: https://github.com/realm/realm-kotlin/blob/main/packages/cinterop/build.gradle.kts#L796, so I do not think you need to add this plugin for it?